### DISCUSSION TITLE
电脑端之呢个接收、不能发送，显示这样问题怎么版？如下图

### DISCUSSION THREAD

**Discussion by ShinyXY** [localsend/localsend#312] [L-简体中文]
DioError [DioErrorType.other]: SocketException: 远程计算机拒绝网络连接。
 (OS Error: 远程计算机拒绝网络连接。
, errno = 1225), address = 192.168.1.10, port = 51471
Source stack:
#0      DioMixin.fetch (package:dio/src/dio_mixin.dart:488)
#1      DioMixin.request (package:dio/src/dio_mixin.dart:483)
#2      DioMixin.post (package:dio/src/dio_mixin.dart:97)
#3      SendNotifier.startSession (package:localsend_app/provider/network/send_provider.dart:97)
<asynchronous suspension>

**Comment by yajlt** [comment]
检查加密方式，IPHONE上如果关闭加密，将无法接收PC发来的文件。
![image](https://user-images.githubusercontent.com/3997602/231748970-360db549-9ea8-4e29-ab4c-960ca0834bbf.png)

**Comment by 01sv** [comment]
我也遇到了这个问题，估计是软件的原因。之前，在微软商店下载的是可以用的。但现在微软商店剔除了LocalSend，在github.com下载完全不能用。
![image](https://user-images.githubusercontent.com/126076824/232224487-da829747-6873-4a19-bbbc-da4d790568f5.png)

**Comment by ShinyXY** [comment]
首先，感谢大家的回复
我说一些我自己的处理方法，先完全卸载软件，然后在”控制面板\系统和安全\Windows Defender 防火墙\允许应用通过Windows Defender防火墙进行通信“面板中，把Localsend删除掉，然后再安装软件，再检查手机电脑端的设置是否一致，保证连在同一网络。
<img width="466" alt="20230418$截图1" src="https://user-images.githubusercontent.com/102151042/232645684-bc96ae8c-93c4-477f-9480-20898a66aece.png">

**Comment by 01sv** [comment]
首先，感谢开发者的制作。我认为 1.8.0 版本有问题，防火墙怎么设置也没有用，。换成 1.7.0 版本立刻就能用了。
![1 8 (1)_副本](https://user-images.githubusercontent.com/126076824/232705022-a181d6b1-1696-408c-b280-14158c9ca46e.jpg)
![1 8 (2)_副本](https://user-images.githubusercontent.com/126076824/232705053-995da396-14ee-49f9-8724-bf0f1e9e9059.jpg)

换成 1.7.0 版本防火墙怎么设置都能用。
![1 7 (1)_副本](https://user-images.githubusercontent.com/126076824/232705273-56c167c7-562a-4646-8a5c-a82dd6ea8396.jpg)
![1 7  (2)_副本](https://user-images.githubusercontent.com/126076824/232705310-70c0363a-00a1-4a21-857c-4b3eb63dd71e.jpg)
![1 7  (3)_副本](https://user-images.githubusercontent.com/126076824/232705346-6d2d363f-c02f-437b-9727-fe3f315b6f4c.jpg)

**Reply by outpcb** [reply]
端口号太大了。修改为小port，就可以了

**Reply by damieai** [reply]
请问如何修改

**Reply by 987472115** [reply]
在软件的设置中，启用最下方的高级设置复选框，把端口从默认的53317改为一个任意三位数就行

**Reply by firstloading** [reply]
> 端口号太大了。修改为小port，就可以了

神了，确实有用。 我反复修改防火墙设置都无效。最后还是改成三位数端口才有用

**Reply by zhizhimangmang0914** [reply]
> 在软件的设置中，启用最下方的高级设置复选框，把端口从默认的53317改为一个任意三位数就行

大佬，我的是1.17版本，改成三位数已经没用了，请问还有什么别的办法吗
